packages/quantic/cypress/integration/facets/facet/facet.cypress.ts

```diff
@@ -221,23 +202,18 @@ describe('Facet Test Suite', () => {
           selectFirstFacetValue();
           cy.wait(InterceptAliases.UA.Facet.Select);
 
-          checkLastValue(FacetSelectors);
+          Actions.checkLastValue();
         }
 
-        describe('verify rendering', () => {
-          before(selectLastFacetValue);
+        it('should render correctly', () => {
+          selectLastFacetValue();
 
           Expect.clearFilterContains('Clear 2 filters');
           Expect.numberOfSelectedCheckboxValues(initialNumberOfSelectedValues);
           Expect.numberOfIdleCheckboxValues(
             defaultNumberOfValues - initialNumberOfSelectedValues
           );
-        });
-
-        describe.skip('verify analytics', () => {
-          before(selectLastFacetValue);
-
-          Expect.logFacetSelect(defaultField, 1);
+          //Expect.logFacetSelect(defaultField, 1);
```

Comment:
Why is this line commented out?

Reply:
I assume to mimick the `.skip` though this test should work with the refactor right?

Reply:
Makes sense. If this is the reason, I'd suggest to add a comment above to clearly indicate it. Adding a reference to the upcoming Jira would be very nice. I think we could handle it in SFINT-4177.